docs/pages/blog/nextgen-pickers.md

```diff
@@ -0,0 +1,130 @@
+---
+title: A fresh approach for date and time inputs
+description: Check out what's new with the next-gen pickers in earlier v6 versions.
+date: 2022-12-20T00:00:00.000Z
+authors: ['josefreitas']
+tags: ['MUI X', 'News']
+---
+
+There's a lot of exciting news in MUI X v6, but there's hardly anything comparable to the revamp we're delivering for the [Date and Time Pickers](https://next.mui.com/x/react-date-pickers/getting-started/).
+
+It's the result of our research and effort to improve these inputs' usability and overall customizability.
+We kept the best aspects of the previous versions, and we're fleshing out the package with several improvements and one fundamental change: a new input type to replace the masked text fields.
+
+## The new date and time fields
+
+This new input type we're talking about is a set of components shortly named [Fields](https://next.mui.com/x/react-date-pickers/fields/).
+They are the base for the new pickers and a revolution in terms of keyboard usability.
```

Comment:
```suggestion
These new input components are called [Fields](https://next.mui.com/x/react-date-pickers/fields/), and they're revolutionary when it comes to navigating through the Picker components with a keyboard.
```

Reply:
Yeah, that's a good point. "Revolutionary" feels like too much of an exaggeration. Maybe something like this instead?

```suggestion
These new input components are called [Fields](https://next.mui.com/x/react-date-pickers/fields/), and they offer massive improvements for navigating through the Picker components with a keyboard.
```

Reply:
In term of keyboard navigation, I think you could say that it's x10 better, given that the mask input had no real keyboard navigation.

But for the component as a whole, I do agree that "revolution" might put the bar a little to high.

Reply:
Ah ok, I misinterpreted what's so great about them. 😁

How about this?

```suggestion
These new input components are called [Fields](https://next.mui.com/x/react-date-pickers/fields/), and they provide massive improvements to the user experience of editing the Pickers with a keyboard.
```

Reply:
That's fine for me.

Strictly speaking the Field being usable on their own, the following is maybe more correct:

```suggestion
These new input components are called [Fields](https://next.mui.com/x/react-date-pickers/fields/), and they provide massive improvements to the user experience of editing the value with a keyboard.
```

Reply:
> In term of keyboard editing, I think you could say that it's x10 better, given that the mask input had no keyboard editing apart from letters and digits.

@flaviendelangle Ok fair, but I think that the sentiments stay, to me, revolutionary would involve turning things around. But I feel that we are taking steps toward an existing non-controversial UX pattern. The latest proposal sounds great.